Exercise_4/docs/part-1.md

```diff
@@ -1,122 +1,306 @@
+    |db56e012-d085-bc77-be8d-b2cb7540f813|1994-11-07|         |999-40-5557|S99975480|X18819519X|Mrs.  |Keisha459 |Walsh511    |      |Heidenreich818|M      |asian|nonhispanic|F     |Huntington Beach  California  US|227 Abernathy Boulevard Unit 19|San Francisco|California|San Francisco County |94111|37.78431771268959 |-122.44773848994711|937646.90          |99.97              |
+    |cc1c0787-b215-3ce4-ef57-41c4b7169434|1973-08-27|         |999-31-9860|S99990549|X13756124X|Mr.   |Dana512   |Nitzsche158 |      |              |M      |white|hispanic   |M     |American Canyon  California  US |787 Ernser Walk                |Santa Clarita|California|Los Angeles County   |91355|34.4617083155669  |-118.56212511979317|1740481.36         |8317.76            |
+    |d4321c71-bdaa-4085-61a1-43cfd876afdc|1990-11-19|         |999-90-7254|S99919958|X69927630X|Mr.   |Frances376|Lynch190    |      |              |M      |asian|nonhispanic|M     |Salinas  California  US         |227 McClure Skyway             |Santa Rosa   |California|Sonoma County        |95403|38.46180438357993 |-122.72061413299186|860907.41          |459.33             |
+    |bf494d2b-8dcf-b448-92c0-a9bb496ad81c|1970-04-16|         |999-57-3297|S99966587|X21870103X|Mr.   |Roberto515|Ferrer164   |      |              |M      |white|hispanic   |M     |Caguas  Puerto Rico  PR         |117 Gleason Mission            |Long Beach   |California|Los Angeles County   |90713|33.825405425893045|-118.16523195240173|1294493.97         |4725.56            |
+    |19543ef9-644c-1c9d-d0d5-e0fcec4670da|1985-06-11|         |999-90-3002|S99931470|X2800981X |Mrs.  |Omega729  |Trantow673  |      |Hessel84      |M      |white|hispanic   |F     |Torrance  California  US        |318 Schaden Manor              |Phelan       |California|San Bernardino County|92371|34.434310800795046|-117.5878097238367 |1981802.53         |22848.21           |
+    |e5bffa4c-f113-e154-28ec-5dd5a51efc74|1999-06-20|         |999-10-3386|S99980475|X214587X  |Ms.   |Raeann749 |Hauck852    |      |              |       |white|nonhispanic|F     |Sausalito  California  US       |411 Feil Well Suite 13         |Malaga       |California|Fresno County        |     |36.65117620339231 |-119.74924813381965|904124.12          |4949.56            |
+    |43a16fb4-b96c-b481-4fa0-c9c39c130224|1957-08-12|         |999-57-9595|S99953108|X81604053X|Mrs.  |Jolanda321|Flatley871  |      |Zieme486      |M      |white|nonhispanic|F     |Rocklin  California  US         |541 Wintheiser Avenue Unit 42  |Beaumont     |California|Riverside County     |92223|33.781281235931075|-116.90570546282912|1630601.76         |428479.92          |
```

Comment:
```suggestion
??? example "Patients Table"
      |Id                                  |BIRTHDATE |DEATHDATE|SSN        |DRIVERS  |PASSPORT  |PREFIX|FIRST     |LAST        |SUFFIX|MAIDEN        |MARITAL|RACE |ETHNICITY  |GENDER|BIRTHPLACE                      |ADDRESS                        |CITY         |STATE     |COUNTY               |ZIP  |
      |------------------------------------|----------|---------|-----------|---------|----------|------|----------|------------|------|--------------|-------|-----|-----------|------|--------------------------------|-------------------------------|-------------|----------|---------------------|-----|
      |42574859-0c77-3889-9b2e-d5476bd0cbd6|2017-08-04|         |999-20-7307|         |          |      |Torie475  |Effertz744  |      |              |       |black|nonhispanic|F     |Concord  California  US         |585 Olson Neck Apt 22          |Oakley       |California|Contra Costa County  |94561|
      |38b9dfb6-29f6-60c8-ef37-6210064feac0|2001-12-22|         |999-72-8255|S99955340|X28520028X|Ms.   |Micaela928|Castañeda925|      |              |       |white|hispanic   |F     |Roseau  Saint George  DM        |905 Kuvalis Lane Suite 6       |Los Angeles  |California|Los Angeles County   |90077|
      |cc837cec-8d1b-31d3-ff1b-4554b32bce35|2002-10-04|         |999-71-4536|S99991837|          |Ms.   |Lorinda137|Ankunding277|      |              |       |white|nonhispanic|F     |Glendora  California  US        |1040 Hintz Fork Suite 42       |Alpaugh      |California|Tulare County        |93201|
      |db56e012-d085-bc77-be8d-b2cb7540f813|1994-11-07|         |999-40-5557|S99975480|X18819519X|Mrs.  |Keisha459 |Walsh511    |      |Heidenreich818|M      |asian|nonhispanic|F     |Huntington Beach  California  US|227 Abernathy Boulevard Unit 19|San Francisco|California|San Francisco County |94111|
      |cc1c0787-b215-3ce4-ef57-41c4b7169434|1973-08-27|         |999-31-9860|S99990549|X13756124X|Mr.   |Dana512   |Nitzsche158 |      |              |M      |white|hispanic   |M     |American Canyon  California  US |787 Ernser Walk                |Santa Clarita|California|Los Angeles County   |91355|
      |d4321c71-bdaa-4085-61a1-43cfd876afdc|1990-11-19|         |999-90-7254|S99919958|X69927630X|Mr.   |Frances376|Lynch190    |      |              |M      |asian|nonhispanic|M     |Salinas  California  US         |227 McClure Skyway             |Santa Rosa   |California|Sonoma County        |95403|
      |bf494d2b-8dcf-b448-92c0-a9bb496ad81c|1970-04-16|         |999-57-3297|S99966587|X21870103X|Mr.   |Roberto515|Ferrer164   |      |              |M      |white|hispanic   |M     |Caguas  Puerto Rico  PR         |117 Gleason Mission            |Long Beach   |California|Los Angeles County   |90713|
      |19543ef9-644c-1c9d-d0d5-e0fcec4670da|1985-06-11|         |999-90-3002|S99931470|X2800981X |Mrs.  |Omega729  |Trantow673  |      |Hessel84      |M      |white|hispanic   |F     |Torrance  California  US        |318 Schaden Manor              |Phelan       |California|San Bernardino County|92371|
     |e5bffa4c-f113-e154-28ec-5dd5a51efc74|1999-06-20|         |999-10-3386|S99980475|X214587X  |Ms.   |Raeann749 |Hauck852    |      |              |       |white|nonhispanic|F     |Sausalito  California  US       |411 Feil Well Suite 13         |Malaga       |California|Fresno County        |     |
      |43a16fb4-b96c-b481-4fa0-c9c39c130224|1957-08-12|         |999-57-9595|S99953108|X81604053X|Mrs.  |Jolanda321|Flatley871  |      |Zieme486      |M      |white|nonhispanic|F     |Rocklin  California  US         |541 Wintheiser Avenue Unit 42  |Beaumont     |California|Riverside County     |92223|
```

Deleted the last few columns, as they are not in an EHR and it will confuse or alarm some people (but this data is generated as part of Synthea).